HW02_A_Graph-Fails.Rmd

```diff
@@ -70,6 +113,27 @@ ggplot(mapping = aes(x = displ, y = hwy, color = drv)) +
   geom_point() + 
   geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
 ```
+#CORRECTED CODE for Graph 4
+This person is getting 3 lines because the variable drv was assigned to a color aesthetic (showing different colors) and since there were three categories, R creates 3 lines, one for each category instead of just one line that shows the general relationship. 
+
+At first, I wasn't sure how exactly I can correct this but I found a post on stackoverflow that was really helpful!
+
+So there are two ways to solve this problem:
+
+1) Override the color aesthetic in geom_smooth layer
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
+  geom_point() + 
+  geom_smooth(se = F, color = "black") #se = F makes it so it won't show the error in the line of fit
+```
+2) Move the color aesthetic only to layer(s) that need it
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point(aes(color = drv)) + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```

```

Comment:
This is a great way to do this!  I couldn't figure this out and when I tried looking it up the answers were all much more complicated than this.